pkg/kubectl/resource_printer.go

```diff
@@ -220,15 +224,15 @@ func (h *HumanReadablePrinter) validatePrintHandlerFunc(printFunc reflect.Value)
 		return fmt.Errorf("invalid print handler. %#v is not a function", printFunc)
 	}
 	funcType := printFunc.Type()
-	if funcType.NumIn() != 4 || funcType.NumOut() != 1 {
+	if funcType.NumIn() != 5 || funcType.NumOut() != 1 {
 		return fmt.Errorf("invalid print handler." +
-			"Must accept 3 parameters and return 1 value.")
+			"Must accept 5 parameters and return 1 value.")
```

Comment:
OK. Please put the expected method signature in a comment on `validatePrintHandlerFunc` and replace the outdated method signature in the comment on line 205 to be "see validatePrintHandlerFunc for required method signature.

I don't know if you tried it or not, but is there a reason `printFunc` is an interface instead of an explicit function pointer type? Seems silly to do this check by hand if the compiler can do it.